src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -142,6 +151,7 @@ ucs_status_t uct_tcp_sockcm_cm_ep_conn_notify(uct_ep_h ep)
     hdr->status          = UCS_OK;
     cep->comm_ctx.length = sizeof(*hdr) + hdr->length;
 
+#if ENABLE_DEBUG_DATA
```

Comment:
instead of ifdef: add function `char *uct_tcp_sockcm_cm_ep_peer_addr_str(cep,buf,max)` and pass it as argument to ucs_trace() macro